test/dotnet-new3.UnitTests/CommonTemplatesTests.cs

```diff
@@ -261,6 +261,7 @@ public void AllCommonItemsCreate(string expectedTemplateName, string templateSho
         [InlineData("LangVersion", null, "Class Library", "classlib", null, null)]
         [InlineData("TargetFramework", "net6.0", "Class Library", "classlib", null, null)]
         [InlineData("Nullable", null, "Class Library", "classlib", null, "netstandard2.0")]
+        [InlineData("Nullable", "enable", "Class Library", "classlib", null, "netstandard2.1")]
```

Comment:
Are we testing if nullable option is ignored here?

Reply:
We are testing if nullable is enabled in csproj file by default. There is no option anymore - it's enabled always for netstandard2.1 and net6.0.